templates/repo/migrate/migrating.tmpl

```diff
@@ -36,10 +36,11 @@
 								<div class="divider"></div>
 								<div class="item">
 									{{if .Failed}}
-									<button class="ui basic red show-modal button" data-modal="#delete-repo-modal">{{.locale.Tr "repo.settings.delete"}}</button>
+										<button class="ui basic red show-modal button" data-modal="#delete-repo-modal">{{.locale.Tr "repo.settings.delete"}}</button>
 									{{else}}
-									<button class="ui basic red show-modal button" data-modal="#cancel-repo-modal">{{.locale.Tr "cancel"}}</button>
+										<button class="ui basic red show-modal button" data-modal="#cancel-repo-modal">{{.locale.Tr "cancel"}}</button>
 									{{end}}
+									<button id="repo_migrating_retry" task-retry-url="{{.Link}}/settings/migrate/retry" class="ui basic button gt-hidden">{{.locale.Tr "retry"}}</button>
```

Comment:
Non-standard attributes (especially for data-passing purpose) should have "data-" prefix.